File: week-07/day-2/Workshop 16.5.2023/sum.py
# Write a function called `sum()` that returns the sum of numbers
# from zero to the given parameter
#
# Example: sum(5) = 0 + 1 + 2 + 3 + 4 + 5 = 15
import sys


def sum(number):
    x = 0
    y = 0
    while x <= number:
        y = x + y
        sys.stdout.write(str(x))
        x += 1
        if x<=number:
            sys.stdout.write(" + ")
    print(" = " + str(y))

sum(5)

File: week-02/day-3/2023.04.12/src/Sum.java
public class Sum {
    public static void main(String[] args) {
        // Create the usual class wrapper (Sum) and main method on your own

// Write a function called `sum()` that returns the sum of numbers from zero
// to the given parameter
//
// Example: sum(5) = 0 + 1 + 2 + 3 + 4 + 5 = 15


        sum(5);
    }
    public static void sum (int num){
        int sum = 0;
        for (int i = 0; i <= num; i++){
            sum = i+sum;

            if (i == 0) {
                System.out.print(i);
            } else {
                System.out.print(" + " +i);
            }
            if (i == num) {
                System.out.println(" = " + sum);
            }
            }
        }
}
